OsmAnd/res/values/strings.xml

```diff
@@ -13,38 +13,38 @@
+    <string name="service_is_not_available_please_try_later">Unavailable service. Please try again later.</string>
+    <string name="mapillary_menu_filter_description_new">Filter images by date or by type in closeup zoom.</string>
     <string name="start_point">Start</string>
-    <string name="remove_group_from_markers">Remove group from markers list</string>
-    <string name="add_group_to_markers">Add group to markers list</string>
-    <string name="routeInfo_roadClass_description">Colouring the route or track line according to road classification.</string>
-    <string name="routeInfo_surface_description">Provide additional information about the physical surface of roads/footpaths.</string>
-    <string name="routeInfo_smoothness_description">Provides a classification scheme regarding the physical usability of a way for wheeled vehicles, particularly regarding surface regularity/flatness.</string>
-    <string name="routeInfo_winter_ice_road_description">Coloring the route or track line according to the winter road classification that is only usable during the winter.</string>
-    <string name="routeInfo_tracktype_description">Coloring a route or track line according to the condition of the roads based on their surface composition. Typically applied to roads where the main road network consists of unpaved roads.</string>
-    <string name="no_items_of_type_message">You do not have any items of this type</string>
-    <string name="no_activity_for_intent">No suitable applications to perform this action were found</string>
+    <string name="remove_group_from_markers">Remove group from list of markers</string>
+    <string name="add_group_to_markers">Add group to list of markers</string>
+    <string name="routeInfo_roadClass_description">Color route or track line by road classification.</string>
+    <string name="routeInfo_surface_description">Additional info about the physical surface of roads/footpaths.</string>
+    <string name="routeInfo_smoothness_description">Classification of physical surface usability for wheeled vehicles, particularly regularity/flatness.</string>
+    <string name="routeInfo_winter_ice_road_description">Color route or track line by winter road classification in winter season.</string>
+    <string name="routeInfo_tracktype_description">Color route or track line according to the condition of the roads based on their surface composition. Typically applied to roads where the main road network consists of unpaved roads.</string>
+    <string name="no_items_of_type_message">You have no items of this type</string>
+    <string name="no_activity_for_intent">Install an app that can perform this action first</string>
     <string name="empty_category_name">Empty category name</string>
-    <string name="amz_manage_subscription_descr">To manage your subscription use the Amazon App on the device or open the Amazon.com website under your profile.</string>
```

Comment:
Verb in front of sentence is used elsewhere. "App" is wrong. People understand what a website is, "Open ___ under your profile" is a strange sentence.